src/main/java/edu/greenblitz/robotName/subsystems/Intake/IntakeConstants.java

```diff
@@ -4,6 +4,7 @@
 
 public class IntakeConstants {
 	public static final int INTAKE_ID = 0;
+	public static final double POWER_TO_RUN = 0;
 	
 	public static class SimulationConstants {
```

Comment:
make SimulationConstants in a different file like in the SwerveModules subsystem